### ISSUE TITLE
Action with multiple paths in filter_path fails

### ISSUE THREAD

**Issue by kargakis** [crytic/slither-action#17]
Not sure why this happens but it seems the action was successful in a PR but fails in master.

PR: https://github.com/regen-network/toucan-bridge/pull/12
Post-merge job fails with Slither trying to analyze path that should be ignored: https://github.com/regen-network/toucan-bridge/runs/7069015805?check_suite_focus=true

**Comment by elopez** [comment]
Hi @kargakis, thanks for the report! This is indeed odd at first sight. It might be related to a slither issue where using the comma separator in a filter is slightly buggy (see eg. crytic/slither#769 crytic/slither#788 crytic/slither#1144). I believe this should be fixed once crytic/slither#1196 lands and 0.8.4 is released.

As a workaround, can you try specifying the expression as a regexp instead of using commas to separate? Something like this in the slither config might help:

```json
	"filter_paths": "(node_modules/|contracts/toucan-contracts/)",
```

**Comment by kargakis** [comment]
Hi @elopez thanks for the recommendation, it worked fine!

https://github.com/regen-network/toucan-bridge/pull/17

I'll close the issue, feel free to re-open if you need to track it.
